### PR TITLE
Tools: Tune: TDFB: Add generate of few needed blobs

### PR DESCRIPTION
**Pull request by singalsu**
See commit texts for descriptions.

### CODE REVIEW

**Review by lgirdwood — COMMENTED** [review]
Can we have a ```sof`-`` prefix for all tools and put the tools in the module directories.

**Comment by singalsu** [comment]
> Can we have a ```sof`-`` prefix for all tools and put the tools in the module directories.

Yep, maybe I should postpone this and do the tools move first. There will be a bit more blobs add later, e.g. a blob for analog mic DRC.

**Comment by kv2019i** [comment]
Release reminder - one week to v2.11-rc1.

**Comment by kv2019i** [comment]
FYI @singalsu , pushing to v2.12.

File: src/audio/tdfb/tune/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Settings blobs generators for TDFB
+
+This directory contains the scripts to generate settgings blobs for
```
Comment (kv2019i) [inline src/audio/tdfb/tune/README.md:3]: typo: settgings

**Review by kv2019i — APPROVED** [review]
Excellent inline docs and README file on top! No blockers.

**Review by lgirdwood — APPROVED** [review]

File: src/audio/tdfb/tune/README.md (hunk 1 above)
Comment (singalsu) [reply src/audio/tdfb/tune/README.md:3]: Oops, fixed now!

**Review by lgirdwood — APPROVED** [review]

**Comment by kv2019i** [comment]
sof-docs fail and Intel LNL fails all known and tracked in https://github.com/thesofproject/sof/issues?q=is%3Aissue+is%3Aopen+label%3A%22Known+PR+Failures%22+

**Pull request merged**